runner/src/main.rs

```diff
@@ -305,22 +305,18 @@ fn build_server(opt: &BuildServer) -> Step {
                                 format!("--target={}", opt.server_rust_target.as_deref().unwrap_or(DEFAULT_SERVER_RUST_TARGET)),
```

Comment:
Is `--no-default-features` still needed here (1 line above)?

Reply:
Not strictly. I've kept it for the emphasis.

Reply:
Up to you, though it seems misleading to me, since default features is already empty (and at a first read this seems to suggest it is not).